python/lsst/analysis/ap/apdbCassandra.py

```diff
@@ -157,7 +157,7 @@ def load_sources_for_object(
             ApdbTables.DiaSource,
             columns=column_names,
             where='"diaObjectId" = ?',
-            limit=limit,
+            limit=int(limit),
```

Comment:
Just curious, is there a case when the argument passed is not an integer? Should the caller be fixed too?

Reply:
Hmm, not sure I understand this. `limit` is an argument to this method, so it must be a caller that passes that `float`. Anyways, I'm fine with `int()` here as an additional protection.